backends/cadence/aot/compiler.py

```diff
@@ -53,8 +57,10 @@ def quantize_pt2(
     converted_model = convert_pt2e(prepared_model)
 
     # Get patterns and apply fusion of dq -> op -> q to qop
-    # pyre-fixme[16]: Pyre doesn't get that CadenceQuantizer has a patterns attribute
-    patterns = [q.pattern for q in quantizer.quantizers]
+    patterns = [
+        assert_is_instance(q, CadenceAtenQuantizer).pattern
```

Comment:
Is this a duplicate and not properly stacked with 
https://github.com/pytorch/executorch/pull/4047?

You can use "gh-stack" to help with this in the future